Example/iOS/ViewControllers/AnimationPreviewViewController.swift

```diff
@@ -22,6 +22,11 @@ class AnimationPreviewViewController: UIViewController {
 
   // MARK: Internal
 
+  override func viewDidDisappear(_ animated: Bool) {
+    super.viewDidDisappear(animated)
+    displayLink?.invalidate()
```

Comment:
I noticed this VC / display link was leaking (because `CADisplayLink` retains its `target` until invalidated, apparently)

Reply:
Oh huh, I thought targets were supposed to be weak. Sounds like a CADisplayLink oddity